tests/test_modeling_common.py

```diff
@@ -62,6 +63,15 @@ class ModelTesterMixin:
     test_missing_keys = True
     is_encoder_decoder = False
 
+
+    def _prepare_for_class(self, inputs_dict, model_class):
+        if model_class in MODEL_FOR_MULTIPLE_CHOICE_MAPPING.values():
+            return {
+                k: v.unsqueeze(1).expand(-1, self.model_tester.num_choices, -1).contiguous() for k,v in inputs_dict.items()
+            }
+        return inputs_dict
+
+
     def test_save_load(self):
         config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
```

Comment:
(nit) could we pass `is_multi_choice` right here and have it be cleaner?
Don't feel strongly.

Reply:
We loop on the models after, sadly.